internal/driver/gomobile/driver.go

```diff
@@ -200,15 +204,13 @@ func (d *mobileDriver) Run() {
 					}
 					if !canvas.inited {
 						canvas.inited = true
-						canvas.painter.Init() // we cannot init until the context is set above
+						canvas.Painter().Init() // we cannot init until the context is set above
 					}
 
-					if d.freeDirtyTextures(canvas) {
+					if canvas.FreeDirtyTextures() {
```

Comment:
Is it ok if I change this part to:
```go
if canvas.IsDirty() {
    canvas.FreeDirtyTextures()
    ....
}
```
?
It would be useful if we consider the dirty flag to solve #2205 and also it would be consistent with the way how this is handled in the desktop version.

Reply:
I would think that would be fine. The `freeDiretyTextures` returning a bool was introduced in this PR anyway.

Reply:
> I would think that would be fine.

Great! I will update.

> The freeDiretyTextures returning a bool was introduced in this PR anyway.

Not really, that was the mobile driver implementation, indeed, you can see it in the diff.

Reply:
done